src/main/java/picocli/CommandLine.java

```diff
@@ -8197,6 +8209,7 @@ private static String describe(ArgSpec argSpec, String separator, String value)
                         description = parameters.description();
                         descriptionKey = parameters.descriptionKey();
                         splitRegex = parameters.split();
+                        usageSplitRegex = parameters.usageSplit().equals("") ? splitRegex : parameters.usageSplit();
```

Comment:
Let's just store the annotation value (and we can decide later whether to use `splitRegex` or `usageSplitRegex`):

```
usageSplitRegex = parameters.usageSplit();
```